presto-elasticsearch/src/main/java/io/prestosql/elasticsearch/client/ElasticsearchClient.java

```diff
@@ -228,6 +239,12 @@ private static RestHighLevelClient createClient(ElasticsearchConfig config, Opti
                 }
             }
 
+            passwordConfig.ifPresent(securityConfig -> {
```

Comment:
verify the two configs are mutually exclusive

Reply:
From this code's point of view, it doesn't matter, I think. Both could be supplied and it should just work.